### PR TITLE
Rake task for re-syncing charges to CRM

### PR DESCRIPTION
**Pull request by anero**

### CODE REVIEW

File: lib/tasks/crm_tasks.rake [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  task :resync_charges => :environment do
+    raise ArgumentError.new('Missing ORGANIZATION_SLUG argument') unless ENV['ORGANIZATION_SLUG'].present?
+
+    organization = Organization.find_by_slug(ENV['ORGANIZATION_SLUG'])
+    raise ArgumentError.new("Organization with slug #{ENV['ORGANIZATION_SLUG']} not found.") if organization.nil?
+
+    charges_to_resync = nil
+    case organization.crm.platform
+    when 'actionkit'
+      charges_to_resync = organization.charges.paid.where('external_id IS NOT NULL')
```
Comment (lavaturtle) [inline lib/tasks/crm_tasks.rake]: Isn't this ones that /have/ been synced to ak?

**Comment by lavaturtle** [comment]
Looks reasonable.  Is this airbrake error related? https://changesprout.airbrake.io/projects/112761/groups/1515933613362222561/notices/1515933983761208808

File: lib/tasks/crm_tasks.rake (hunk 1 above)
Comment (anero) [inline lib/tasks/crm_tasks.rake]: Yeah, that's exactly the oposite of what we want :/ updating...

**Comment by anero** [comment]
Yeah, that was me testing with a really old un-synced charge in production, it's probably bad data from the early days of TC

**Pull request merged**